be/src/exec/pipeline/result_sink_operator.cpp

```diff
@@ -42,6 +43,9 @@ Status ResultSinkOperator::prepare(RuntimeState* state) {
     case TResultSinkType::VARIABLE:
         _writer = std::make_shared<VariableResultWriter>(_sender.get(), _output_expr_ctxs, _profile.get());
         break;
+    case TResultSinkType::HTTP_PROTOCAL:
```

Comment:
is it  a general protocol for all data formats including json/csv/tsv? 
show to distinguish different format?

Reply:
right now just for json. If csv is needed, should we support cast_to_string for complex type like array/map/struct?